### DISCUSSION TITLE
WebSocket connection to 'wss://q.bridge.walletconnect.org/?env=browser FAILED

### DISCUSSION THREAD

**Discussion by outlawcode** [WalletConnect/walletconnect-monorepo#2948] [Developer Support]
Hello,

I got this error, anyone please help?

WebSocket connection to 'wss://q.bridge.walletconnect.org/?env=browser

FAILED

**Comment by xzilja** [comment]
That uri refers to V1 bridge, which was shut down on 28th of June
https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e

Please upgrade to V2 following one of the paths at
https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/overview

**Reply by outlawcode** [reply]
Hello,

How to create my own bridge server?
My app is too large, can not migrate from v1 to v2 in one or two days.

Thank you!

**Reply by finessevanes** [reply]
What SDK are you using?

**Comment by outlawcode** [comment]
> What SDK are you using?

Currently web3 modal and @walletconnect/web3-provider
